downstream/modules/builder/con-container_file.adoc

```diff
@@ -7,7 +7,8 @@ Once your definition file is created, Ansible Builder reads and validates it, th
 . Fetch base image
```

Comment:
```suggestion
. Fetch base image.
```

Reply:
This isn't a task, but rather a description of the steps that Ansible builder performs to create the container image. Leaving as is.